x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/inference_info.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiSpacer, EuiCallOut, EuiText } from '@elastic/eui';
+
+interface Props {
+  inferenceType: string;
+  info: string[];
+}
+
+export const getInferenceInfoComponent = (inferenceType: string, info: string[]) => (
+  <InferenceInfo inferenceType={inferenceType} info={info} />
+);
+
+export const InferenceInfo: FC<Props> = ({ inferenceType, info }) => {
+  if (info.length === 0) {
+    return null;
+  }
+  return (
+    <>
+      <EuiCallOut
+        title={i18n.translate('xpack.ml.editModelSnapshotFlyout.calloutTitle', {
+          defaultMessage: 'Model type: {inferenceType}',
```

Comment:
My understanding is that models can be used for multiple types of tasks, so it's not the model type that's important but rather the task that you specified when you imported it (e.g. via the --task-type option in the [eland_import_hub_model](https://www.elastic.co/guide/en/elasticsearch/client/eland/current/machine-learning.html#ml-nlp-pytorch) script) 
```suggestion
          defaultMessage: 'Task: {inferenceType}',
```

Reply:
Updated in https://github.com/elastic/kibana/pull/135566/commits/3c5ad088a1aa1fc738a14edacec8182bb55430b9